src/modm/math/algorithm/prescaler.hpp

```diff
@@ -25,14 +26,15 @@ namespace modm
  * @note For ranges the end is *inclusive*: [begin, end]!
  * @ingroup	modm_math_algorithm
  */
+template<typename T>
```

Comment:
```suggestion
template<std::unsigned_integral T>
```

Reply:
with `#include <concepts>`

Reply:
you can probably remove the `#include <tuple>` I'm not sure what it's used for…

Reply:
`#include <stdint.h>` can be the C++ version of that header `#include <cstdint>`

Reply:
Also `#include <initializer_list>` was missing and the reason why some STM32F0 builds failed in the CI.